music21/pitch.py

```diff
@@ -2912,6 +2912,10 @@ def pitchClass(self) -> int:
         >>> a.pitchClass = 1
         >>> a
         <music21.pitch.Pitch C#3(+20c)>
+
+        OMIT FROM DOCS
+
+        this is a performance critical operation.
         '''
         pc = round(self.ps) % 12
         if pc == 12:
```

Comment:
speaking of performance, what is `if pc == 12` doing here? was modulo different in Python 2? 🤔

Reply:
OMG -- I totally forgot about this bug.  Yes!  It was possible before Python 2.5 to have negative ps's become 12 so we had to do more complex things to fix it.  Totally impossible for years now!  But we need to cast it to an int -- so there IS a bug here.

Reply:
No wait, it doesn't need to cast to an int.  That's only because I was playing around in Python 2 just now.  Good riddance!